scalafix-core/src/main/scala/scalafix/internal/patch/ReplaceSymbolOps.scala

```diff
@@ -19,6 +21,20 @@ object ReplaceSymbolOps {
       case _ => None
     }
   }
+  private def extractImports(stats: Seq[Stat]): Seq[Import] = {
+    stats
+      .takeWhile(_.is[Import])
+      .collect { case i: Import => i }
+  }
+
+  @tailrec private final def getGlobalImports(ast: Tree): Seq[Import] =
```

Comment:
this could be more granular and collect relevant scoped imports, all the way to the global ones at the top